### PR TITLE
[AVAX] Merge Cortina 2 (v1.10.2)

### PR DESCRIPTION
**Pull request by evlekht**
## Why this should be merged
This PR contains changes from ava-labs/avalanchego between cortina-1 and cortina-2.
### Conflicts
All gh workflows and .md are remain unchanged in favor of our version.
- RELEASES.md
- config/config.go
- go.mod
- go.sum
- scripts/constants.sh
- scripts/lint.sh
- version/compatibility.json
- version/constants.go
- vms/platformvm/vm.go
### Post-merge fixes
- Unit-tests `testNetworkID` var was removed, so its replaced with `constants.UnitTestID` and HRP replaced with `constants.UnitTestHRP`
- Various small changes to fit rules enforced by new linters.
- Reverted our changes to how linter is run. Its slower, but results are easier to read and navigate and are more reliable.
- Reverted avax changes to license headers handling, cause new license linter can't fully support different license headers. Currently, no license header linter is used (old one wasn't working for some time, needs investigation), this will be fixed later.
## How this was tested
With already existing unit-tests and e2e tests.

### CODE REVIEW

File: version/compatibility.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "26": [
+    "v1.1.2",
```
Comment (knikos) [inline version/compatibility.json:3]: have we missed the in-between version of 1.1.1?

**Review by knikos — COMMENTED** [review]
have you checked what these state management changes are about . I see that `vms/platformvm/vm.go` is heavily changed.

File: version/compatibility.json (hunk 1 above)
Comment (evlekht) [reply version/compatibility.json:3]: unfortunately, I didn't update version and compatibility in code in cortina 1 PR, so we can't have them here
but starting form cortina 2, we'll keep this stuff consistent

**Comment by evlekht** [comment]
> have you checked what these state management changes are about . I see that `vms/platformvm/vm.go` is heavily changed.

Yes, they segregated some code, it was moved to `vms/platformvm/validators/manager.go`. I'm not 100% agree with this change, cause some stuff doesn't seem to belong to new place, but nothing harmful here.

**Review by knikos — APPROVED** [review]

**Pull request merged**